mysql/datadog_checks/mysql/statements.py

```diff
@@ -1,6 +1,7 @@
 # (C) Datadog, Inc. 2020-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
+import json
```

Comment:
I think this is a mistaken import

Reply:
Oh whoops, was debugging something. Removed